hcx-onboard/src/main/java/org/swasth/hcx/services/ParticipantService.java

```
@@ -124,9 +124,13 @@ private void createParticipantAndSendOTP(HttpHeaders headers, OnboardRequest req
             participant.put(SCHEME_CODE, "default");
         String identityVerified = PENDING;
         if (ONBOARD_FOR_PROVIDER.contains(request.getType())) {
-            identityVerified = identityVerify(headers, getApplicantBody(request));
-            if (StringUtils.equalsIgnoreCase(identityVerified, REJECTED))
-                throw new ClientException("Identity verification is rejected by the payer, Please reach out to them.");
+            String query = String.format("SELECT * FROM %s WHERE applicant_email ILIKE '%s' AND status IN ('Pending', 'rejected')", onboardingTable, request.getPrimaryEmail());
```

Use constants instead of hardcoding pending and rejected